toolbox/providers/gcp/gcp-advanced.md

```diff
@@ -126,21 +138,23 @@ Options:
 - `service-account-config-file` `[gcp-service-account.json]` Path to GCP service account config file.
 - `platform-config-file` Path to platform config file
 
-### Use cases
+### Use Cases
 - #### Destroy cluster
 ```$xslt
 ./gcp.sh --k8s-cluster nauta-cluster --operation destroy
 ```
 
-- #### Create cluster
+- #### Create Cluster
+
 This command creates gke cluster and resources related to the cluster (gateway, nfs).
```

Comment:
Shouldn't this be Execute this command to create a GKE cluster and resources related to the cluster (gateway, NFS) so that it matches our documents (OS/ENTERPRISE Install guide)?